python/ray/rllib/agents/a3c/__init__.py

```diff
@@ -1,4 +1,10 @@
-from ray.rllib.agents.a3c.a3c import A3CAgent, DEFAULT_CONFIG
-from ray.rllib.agents.a3c.a2c import A2CAgent
+from ray.rllib.agents.a3c.a3c import A3CTrainer, DEFAULT_CONFIG
+from ray.rllib.agents.a3c.a2c import A2CTrainer
 
-__all__ = ["A2CAgent", "A3CAgent", "DEFAULT_CONFIG"]
+# TODO: remove the legacy agent names
+A2CAgent = A2CTrainer
```

Comment:
consider something like - or else it will fail silently when you end up removing these agent names.
```python
def A2CAgent(*args, **kwargs):
    logger.warning("This will be removed - giving back an A2CTrainer") # change text
    return A2CTrainer(*args, **kwargs)
```

Reply:
Done